callmetadata.schema.json

```diff
@@ -7,8 +7,37 @@
     "definitions": {
         "participant": {
             "type": "object",
-            "description": "Specific details about the farEndUsers (the other end of the call)",
+            "description": "Specific details about a participant of the call",
+            "additionalProperties": true,
             "properties": {
+                "name": {
+                    "type": "string",
+                    "example": "Steve Jones",
+                    "description": "Full name of the participant on the call"
+                },
+                "address": {
+                    "type": "string",
+                    "example": "sip:9991234567@voip-provider.net",
+                    "description": "The address of the participant"
+                },
+                "startTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "example": "2016-06-18T07:50:30Z",
+                    "description": "Date and time when the participant was requested to participate on the call"
+                },
+                "joinTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "example": "2016-06-18T07:50:30Z",
+                    "description": "Date and time when the participant joined the call"
+                },
+                "endTime": {
```

Comment:
Maybe `exitTime` will be a better name for this attribute after we've changed description.

Reply:
I agree, good observation.  Since the call has a life outside this leg, and it has symmetry with 'join' above.  I'd say this is a mandatory change.